operator/validator/controller.go

```diff
@@ -969,13 +990,21 @@ func (c *controller) UpdateValidatorMetaDataLoop() {
 	var sleep = 2 * time.Second
 
 	for {
+		activeSubnets := c.network.ActiveSubnets()
+
 		// Get the shares to fetch metadata for.
 		start := time.Now()
 		var existingShares, newShares []*ssvtypes.SSVShare
 		c.sharesStorage.Range(nil, func(share *ssvtypes.SSVShare) bool {
 			if share.Liquidated {
 				return true
 			}
+
+			subnet := networkcommons.CommitteeSubnet(share.CommitteeID())
```

Comment:
you don't want to fetch metadata for a share if its active? what if the validator's status suddenly changes?

Reply:
fixed